src/DGtal/topology/Object.ih

```diff
@@ -60,7 +61,12 @@
 template <typename TDigitalTopology, typename TDigitalSet>
 inline
 DGtal::Object<TDigitalTopology, TDigitalSet>::Object()
-  : myTopo( 0 ), myPointSet( 0 ), myConnectedness( UNKNOWN )
+  : myTopo( nullptr ),
```

Comment:
I can imagine! I digged directly into c++11 though. [Here](http://stackoverflow.com/questions/13816385/what-are-the-advantages-of-using-nullptr) says it solves overloading problems because nullptr is a pointer type, where 0 or NULL are int types.